openfn/importer/workflows/example/project.yaml

```
@@ -0,0 +1,1310 @@
+            body: JSON.stringify($.issue),
+            headers: { 'content-type': 'application/json' },
+          });
+
+          fn(state => {
+            console.log('Sending failed bundle to kafka topic dead-queue...');
+            return state;
+          });
+
+          //send final Bundle we mapped
+          post('/openfn/dead-queue', {
+            body: $.bundle, 
```

yamllint

[error] trailing spaces

(trailing-spaces)